code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -519,3 +519,177 @@ var/global/image/plasmaman_on_fire = image("icon"='icons/mob/OnFire.dmi', "icon_
 		return 0
 	return 1
 
+
+
+
+var/global/list/synth_flesh_disguises = list()
+
+/datum/species/synth
+	name = "Synth" //inherited from the real species, for health scanners and things
+	id = "synth"
+	say_mod = "beep boops" //inherited from a user's real species
+	sexes = 0
+	specflags = list(NOTRANSSTING,NOBREATH,VIRUSIMMUNE) //all of these + whatever we inherit from the real species
+	safe_oxygen_min = 0
+	safe_toxins_min = 0
+	safe_toxins_max = 0
+	safe_co2_max = 0
+	SA_para_min = 0
+	SA_sleep_min = 0
+	dangerous_existence = 1
```

Comment:
Yeah this is for stuff that's deadly to be under normal circumstances rather than deadly to be because you're a scary antag.
If there's a code reason for it it's okay I guess though.